documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -161,23 +176,23 @@ For example, consider a SQL Server installation with an `inventory` database tha
 
 === Schema change topic
 
-The user-facing schema change topic is not implemented yet (see {jira-url}/browse/DBZ-753[DBZ-753]).
+The user-facing schema change topic is not implemented yet (see https://issues.jboss.org/browse/DBZ-753[DBZ-753]).
```

Comment:
Should we use issues.redhat.com rather than issues.jboss.org?  If I missed this in other changes in this PR, it would apply across the board.  We recently changed this to `{jira-url}` so we could source that from an attribute, I see no reason why we shouldn't continue that, no?

Reply:
+1